src/components/builder/multiple.tsx

```diff
@@ -22,6 +26,10 @@ function Multiple<T = unknown>() {
 
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     formikOnChange(e);
+
+    // only touch default value if we're allowed to
+    if (!manageDefaultValue) return;
```

Comment:
Maybe `alterDefaultValue`? Not satisfied either with this one, so maybe add a docstring in props to quickly describe what it does

Reply:
went for `updateDefaultValue` + docstring